lib/aerospike.js

```diff
@@ -48,8 +48,7 @@ exports.filter = require('./filter')
  * Client#operate} command.
  * @summary {@link module:aerospike/exp|aerospike/exp} module
  */
-const exp = exports.exp = require('./exp')
-exp.type = as.exp.type
```

Comment:
Won't this break this line further down?

    exp.operations = require('./exp_operations')